src/vue-testing-library.js

```diff
@@ -53,15 +57,12 @@ function render(
   })
 
   mountedWrappers.add(wrapper)
-
-  const div = document.createElement('div')
-  wrapper.element.parentNode.insertBefore(div, wrapper.element)
```

Comment:
I'm not 100% sure this line should be gone. If I add it back, [this inline snapshot](https://github.com/testing-library/vue-testing-library/pull/100/files#diff-b45a2abc5c6c583db5549af5a49a2aa7R19) changes to `<blink></blink>`, and I don't think that's the desired output.

@dfcook could you double check that the logic here is no longer necessary as long as we append `wrapper.element` to our container (which is, in turn, a `<div>` wrapped in a `<body>`)?

Reply:
I agree, given that we are now in control of the container we can remove this logic.